compiler-plugin/src/main/java/arrow/meta/diagnostic/MetaDefaultErrorMessages.java

```diff
@@ -0,0 +1,39 @@
+package arrow.meta.diagnostic;
```

Comment:
Awesome @i-walker :raised_hands: Please, what's the reason of **Java** source for several classes in `compiler-plugin`? I know that Java sources were added in `idea-plugin` because of a copy-paste from other repository (see `README.txt` in `idea-plugin` sources). Is it the same case ? If yes, could we add that repository? Thank you so much!!

Reply:
The Java sources are necessary for the Error Message classes. They have a rather special initializer, which I got only working in Java, but tried to replicate in Kotlin with no success. 
The compiler does the same for `org.jetbrains.kotlin.diagnostics.Errors`.
To resolve that, one would have to change a private member of the DiagnosticFactory, so the NPE doesn't occur at Runtime loading the class. 
It's a bug-fix, I can contribute if that is desired.

Reply:
Great, thanks for the info @i-walker :raised_hands: It would be awesome if you could add a README file like Joachim made in `idea-plugin` to add the link of those sources for future updates.